### PR TITLE
ui: Add Svelte UI scaffolding

### PR DESCRIPTION
**Pull request by juliaogris**
Add Svelte boilerplate for a default svelte app, formatted with prettier
and added to the CI and Docker build, and some node modules used for
development (sirv in particular). Svelte puts static assets in
`frontend/public`, so change the backend to serve that directory instead
of `static/`.

Preview: https://pr24.foxtrot.jul.run

### CODE REVIEW

File: frontend/.gitignore [hunk 1]
```diff
@@ -0,0 +1,4 @@
+/node_modules/
+/public/build/
+
+.DS_Store
```
Comment (camh-) [inline frontend/.gitignore]: I assume this was generated which is why it is here, but it is not later removed. At the very least, this does not belong in a subdirectory gitignore file, but I think we agree it does not belong in a repo's gitignore at all.

File: .dockerignore [hunk 2]
```diff
@@ -0,0 +1,3 @@
+/frontend/node_modules/
+/frontend/bundle.*
```
Comment (camh-) [inline .dockerignore]: We're not using this path pattern yet in the build. I think it comes later.

**Review by camh- — COMMENTED** [review]
Just a couple of little things...

File: frontend/.gitignore (hunk 1 above)
Comment (juliaogris) [reply frontend/.gitignore]: I have added a bigger purge to the follow-up PR. If you want I could add it here.:
https://github.com/foxygoat/foxtrot/pull/25/commits/973beb15b65f1896ee8d4292c000fab190721b13

File: .dockerignore (hunk 2 above)
Comment (juliaogris) [reply .dockerignore]: yes sorry!

Comment (juliaogris) [reply .dockerignore]: Done.

File: frontend/.gitignore (hunk 1 above)
Comment (juliaogris) [reply frontend/.gitignore]: ok, I've moved the commit to this PR.

**Review by camh- — APPROVED** [review]
LGTM :herb:

**Pull request merged**